src/iter/mod.rs

```diff
@@ -561,12 +561,14 @@ pub trait ParallelIterator: Sized {
     /// Note that not all parallel iterators have a useful order, much like
     /// sequential `HashMap` iteration, so "last" may be nebulous.
     fn find_last<P>(self, predicate: P) -> Option<Self::Item>
-        where P: Fn(&Self::Item) -> bool + Sync {
+        where P: Fn(&Self::Item) -> bool + Sync
+    {
         find_first_last::find_last(self, predicate)
     }
 
     #[doc(hidden)]
-    #[deprecated(note = "parallel `find` does not search in order -- use `find_any`, `find_first`, or `find_last`")]
+    #[deprecated(note = "parallel `find` does not search in order -- use `find_any`, \\
```

Comment:
I am uncertain if this is the appropriate way to handle splitting a Single string into a MultiLine String Split to lower 
line length which this exceeded.